moment.parseFormat.js

```diff
@@ -162,6 +166,18 @@
     // check if first < 13 && last > 12, then it must be MM/YY
     format = format.replace(regexMonthYearShort, 'MM/YY')
 
+    // check if both numbers are < 12, then it must be something wrong
+    format = format.replace(regexDayYear, '')
+
+    // to prevent 9.20 gets formated to D.Y, we format the complete date first, then go for the time
+    if (format.match(formatIncludesMonth)) {
```

Comment:
I thought about this again, and I think instead of checking for month, we should check for day, because only if day is provided, a time makes sense, right?

For the regex you can simply use `/D/`. We might make it more sophisticated in future as we need to ignore escaped words like `[ignore this]`, but it's a problem when it's a problem